src/file_manager.hpp

```diff
@@ -28,7 +28,7 @@ namespace big
 
 	private:
 		std::filesystem::path m_base_dir;
-
+		std::filesystem::path m_nav_dir;
```

Comment:
What is this?

Reply:
It was to ensure the navigation directory was there, where the sounds that are used should be placed.